cognite/client/data_classes/transformations/common.py

```diff
@@ -115,53 +115,58 @@ def sequence_rows(external_id: str = "") -> SequenceRows:
     @staticmethod
     def data_model_instances(
         model_external_id: str = "", space_external_id: str = "", instance_space_external_id: str = ""
-    ) -> DataModelInstances:
-        """To be used when the transformation is meant to produce data model instances.
-            Flexible Data Models resource type is on `beta` version currently.
-
+    ) -> DataModel:
+        """
         Args:
             model_external_id (str): external_id of the flexible data model.
             space_external_id (str): space external_id of the flexible data model.
             instance_space_external_id (str): space external_id of the flexible data model instance.
         Returns:
             TransformationDestination pointing to the target flexible data model.
         """
-        return DataModelInstances(
+        return DataModel(
             model_external_id=model_external_id,
             space_external_id=space_external_id,
             instance_space_external_id=instance_space_external_id,
         )
 
     @staticmethod
-    def instance_nodes(view: Optional[ViewInfo] = None, instance_space: Optional[str] = None) -> InstanceNodes:
-        """To be used when the transformation is meant to produce node's instances.
-            Flexible Data Models resource type is on `beta` version currently.
-
+    def nodes(view: Optional[ViewInfo] = None, instance_space: Optional[str] = None) -> Nodes:
```

Comment:
Changing the name and return type of a method are breaking changes. If you want to introduce breaking changes, you should first introduce them in a backwards-compatible way and add deprecation warnings, then we can remove deprecated parts in the next major release.

Reply:
yes, the request is to break the methods and renaming the class DataModel, Nodes, Edges, and Instances. (@SunilVenekarKrishnamoorthy  fyi)